todo_app/templates/index.html

```diff
@@ -21,8 +21,8 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        {% for item in items %}
-        <li>{{ item.id }} - {{ item.name }} - {{item.status}}</li>
+        {% for item in view_model.items %}
+              # {{item.name}}
```

Comment:
You could keep the full `{{ item.id }} - {{ item.name }} - {{item.status}}` that you had before.